File: python/1546.py
'''
    문제 이름 : 평균
    문제 번호 : 1546
    문제 링크 : https://www.acmicpc.net/problem/1546
'''

N = int(input());
result = [];
score = list(map(int,input().split())); #set(map()) 써서 틀렸음

maxScore = max(score);

for i in score:
    result.append(i/maxScore*100);
test_avg = sum(result)/N;
print(test_avg);

File: js/class_1/1546.js
/*
    문제 이름 : 평균
    문제 번호 : 1546
    문제 링크 : https://www.acmicpc.net/problem/1546
*/

const input = require("fs").readFileSync("dev/stdin").toString().split("\n");

const N = parseInt(input[0]);
const grade = input[1].split(" ");
const maxGrade = Math.max(...grade);
let sum = 0;

grade.map((item) => {
  sum += (item / maxGrade) * 100;
});

console.log(sum / N);
